styles/icons.less

```diff
@@ -392,6 +392,7 @@
 .infopath-icon:before      { .fi; content: "\ea35"; top: 2px; font-size: 16px; }
 .inform7-icon:before       { .fi; content: "\e984"; top: 2px; font-size: 16px; .sharpen; }
 .inno-icon:before          { .fi; content: "\e985"; top: 2px; }
+.imba-icon:before					 { .fi; content: "\e90e"; top: 2px; font-size: 16px; }
```

Comment:
Ah, those are actually spaces being used for alignment. My ideal formatting style
1. Tabs to indent
2. Spaces to align
3. Keep the two separated with visible/non-whitespace characters, so there's no ambiguity

Death to StandardJS!

Reply:
So yeah, just replace the tabs on this line with spaces and we're right to merge. :)

Reply:
Oh, and could you map the codepoint to `\1f4a0` instead? The copy of the font I have staged in IcoMoon has the new icon mapped to that, not `\e90e`.

I know, this is a clumsy pipeline. 😞 I intend to automate this properly. Eventually...